Bottom: 6bfa18a292efc9f91c4ff5cd6f5ea273ec8b6583
Top:    718b39d442446811c39ae9f2134ec998fa883ef6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-03 23:54:02 +0000

Refresh of bentley-ottann-test-implementa

---

diff --git a/src/main.c b/src/main.c
index a72dad4..a0b38f3 100644
--- a/src/main.c
+++ b/src/main.c
@@ -40,7 +40,6 @@
 #include <signal.h>
 #include <unistd.h>
 #include <sys/stat.h>
-#include <setjmp.h>
 
 #include "global.h"
 #include "data.h"
@@ -54,7 +53,6 @@
 #include "action.h"
 #include "misc.h"
 #include "lrealpath.h"
-#include "sweep.h"
 
 /* This next one is so we can print the help messages. */
 #include "hid/hidint.h"
